contracts/interfaces/IDnGmxJuniorVault.sol

```diff
@@ -45,15 +44,8 @@ interface IDnGmxJuniorVault is IERC4626, IBorrower {
     event FeesWithdrawn(uint256 feeAmount);
 
     event DepositCapUpdated(uint256 _newDepositCap);
-    event BatchingManagerUpdated(address _batchingManager);
-
-    event AdminParamsUpdated(
-        address newKeeper,
-        address dnGmxSeniorVault,
-        uint256 newDepositCap,
-        address batchingManager,
-        uint16 withdrawFeeBps
-    );
+
+    event AdminParamsUpdated(address newKeeper, address dnGmxSeniorVault, uint256 newDepositCap, uint16 withdrawFeeBps);
```

Comment:
Changing the topic would add some complexity while indexing AdminParamsUpdated event. It'd be helpful if we just emit zero address for batching manager and not change the event.